templates/deployment-web.yaml

```diff
@@ -233,6 +247,22 @@ spec:
           resources:
             {{- toYaml . | nindent 12 }}
           {{- end }}
+        {{- if .Values.mastodon.metrics.prometheus.enabled }}
+        - name: prometheus-exporter
+          image: "{{ coalesce .Values.mastodon.web.image.repository .Values.image.repository }}:{{ coalesce .Values.mastodon.web.image.tag .Values.image.tag .Chart.AppVersion }}"
+          command:
+            - ./bin/prometheus_exporter
```

Comment:
This should probably be `./bin/prometheus_exporter --port {{ .Values.mastodon.metrics.prometheus.port }}` so it listens to the port that is configured (default is `9394`)